src/interpreter/dynamic_loading_ffi/on_demand_compiler.nit

```diff
@@ -45,10 +54,24 @@ redef class AMethPropdef
 end
 
 redef class NaiveInterpreter
+	redef fun start(mainmodule)
+	do
+		super
+
+		# Delete temporary files
+		var compile_dir = compile_dir
+		if compile_dir.file_exists then compile_dir.rmdir
+	end
+
 	# Where to store generated C and extracted code
-	#
-	# TODO make customizable and delete when execution completes
-	private var compile_dir = "nit_compile"
+	private var compile_dir: String is lazy do
+		var opt = modelbuilder.toolcontext.opt_compile_dir.value
+		if opt != null then return opt
+		return "/tmp/niti_ffi_{process_id}"
```

Comment:
I do not like the possible `leaking` information when storing things in a world readable and writable place. Maybe `$XDG_RUNTIME_DIR` ? It is safer than /tmp I think.